packages/@cdktf/hcl2cdk/lib/expressions.ts

```diff
@@ -2,33 +2,24 @@
 // SPDX-License-Identifier: MPL-2.0
```

Comment:
Useful to look at this file commit by commit. First commit basically takes out all the functions that aren't related to TF Expr -> TS, and then the subsequent commits clean up the actual conversion